### DISCUSSION TITLE
How to deploy nodes at different machines in cluster mode?

### DISCUSSION THREAD

**Discussion by rxy1212** [qdrant/qdrant#3089] [Q&A]
How to deploy nodes at different machines in cluster mode with docker?

**Comment by timvisee** [comment]
This is explained here: https://qdrant.tech/documentation/guides/distributed_deployment/

Make sure your machines can talk to each other. Start the first node on a machine (through Docker) with the `--uri` option. Then start the other nodes on the rest of the machines with the `--bootstrap` option pointing to the first.

For Kubernetes we have a Helm Chart available: https://github.com/qdrant/qdrant-helm

Please note that we also offer a Cloud solution, where we do all of this for you: https://qdrant.to/cloud
